### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'new/3dSwap/datasets/CelebA-HD/final_crops/gsy_src_2.jpg'

### ISSUE THREAD

**Issue by qym77** [lyx0208/3dSwap#12]
if i only want to run the order " python run_3dSwap.py" to see the result do i need to download datasets or some thing?because there is an error,readme seems not mention this

**Comment by qym77** [comment]
FileNotFoundError: [Errno 2] No such file or directory: '/new/3dSwap/output/faceswap/gsy_src_2_gsy_tar_1.png'
why output still miss image?should i create a folder to address this?but i think its not the right way to fix it,because i have so many file missing,thank you for u reply

**Comment by takuphilchan** [comment]
@qym77 Hello, yes you need to modify that part and use argument parser to dynamically parse the input and output paths of your images  and their corresponding camera poses, to process you dataset and get camera poses you can check the notebook.  You can update faceswap_coach.py like this and ensure the argument parser expects these inputs Like this 
![Screenshot 2024-07-31 122429](https://github.com/user-attachments/assets/aa6d3507-63e1-4ac0-8220-fe639092331e)

**Comment by qym77** [comment]
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/traffic/home/new/3dSwap/output/faceswap/0_1.png'
thank u for the reply,but what can i do for the output file missing?

**Comment by lyx0208** [comment]
Sorry, there's something wrong with the original facewswap_coach.py file, and I've just updated it.
For the output file, you can first make a directory "output/faceswap" under the main 3dSwap directory.

**Comment by qym77** [comment]
> 抱歉，原始的 facewswap_coach.py​​ 文件有问题，我刚刚更新了它。 对于输出文件，您可以先在主 3dSwap 目录下创建一个目录“output/faceswap”。

yes,thank u for your great work and help! hope i can understand it better！ ;）
